handler.go

```diff
@@ -70,7 +70,10 @@ func createRequestHandler(configuration Configuration) (http.HandlerFunc, error)
 			}
 			return
 		}
-		username := cas.Username(req)
+
+		username := cas.Attributes(req).Get("username")
```

Comment:
Can this lead to potential problems when the username of an already available user changes?

We should also consider checking the attribute `preferred_username` first and only take the `username` attribute when the other one is not set.